certified-connectors/DocumentsCorePack/apiProperties.json

```diff
@@ -1,21 +1,23 @@
-{
-  "properties": {
-    "connectionParameters": {
-      "api_key": {
-        "type": "securestring",
-        "uiDefinition": {
-          "displayName": "API Key",
-          "description": "The API Key for this api",
-          "tooltip": "Provide your API Key",
-          "constraints": {
-            "tabIndex": 2,
-            "clearText": false,
-            "required": "true"
-          }
-        }
-      }
-    },
-    "iconBrandColor": "#d7f4f0",
-    "capabilities": []
-  }
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "Get an API Key - https://www.mscrm-addons.com/getapikey",
+          "tooltip": "Provide your API Key - Get it from https://www.mscrm-addons.com/getapikey",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#d7f4f0",
+    "capabilities": [],
+	"publisher": "PTM EDV Systeme GmbH",
+	"stackOwner": "PTM EDV Systeme GmbH"

```

Comment:
@poweraddons can you please fix the indentation here?

Reply:
Can you explain me whats wrong here?

Reply:
Formatting, the indentation needs to be fixed. You can see when looking at [the whole file](https://github.com/microsoft/PowerPlatformConnectors/blob/718cd9d591fe8d8fd4383849e8634a17c4671c53/certified-connectors/DocumentsCorePack/apiProperties.json) that lines 20 and 21 are too far away and line 19 is at the right place.

Reply:
ok. I uploaded a new apiproperties.json with the fix